app/controllers/posts_controller.rb

```diff
@@ -20,9 +19,29 @@ def like
     end
   end
 
+  def new
+    @post = @user.posts.build
+  end
+
+  def create
```

Comment:
- [ ] Great job working on this milestone. In our config/routes.rb, kindly add a route for creating post.